tools/apiview/parsers/js-api-parser/readme.md

```diff
@@ -12,10 +12,31 @@ This application tokenises a Javascript project into a format useful for JavaScr
 
 Run API extractor step on JS project to create json output file. This step is integrated within build commend for all Azure SDK projects in azure-sdk-for-js monorepo. So running build step is good enough to create input file for APIvIew parser. You can see a JSON file created in temp directory within package root directory once build step is completed successfully for the package.
 
-Run `node ./dist/export.js <Path to api-extractor JSON output> <Path to apiviewFile>`
+Run `node ./dist/export.js <Path to api-extractor JSON output> <Path to apiviewFile> [Path to metadata.json]`
 
 For e.g.
 
 `node .\export.js C:\git\azure-sdk-for-js\sdk\core\core-client\temp\core-client.api.json C:\git\azure-sdk-for-js\sdk\core\core-client\temp\apiview.json` 
 
-Or if you have the package installed, you can run `ts-genapi <Path to api-extractor JSON output> <Path to apiviewFile>`.
\ No newline at end of file
+Or if you have the package installed, you can run `ts-genapi <Path to api-extractor JSON output> <Path to apiviewFile> [Path to metadata.json]`.
+
+### Cross-Language IDs
+
+The optional third parameter allows you to specify a path to a `metadata.json` file that contains cross-language definitions. This enables API reviewers to correlate APIs that are generated from the same service API specification across different languages.
+
+Example usage with metadata file:
+`ts-genapi input.api.json output.json metadata.json`
```

Comment:
is it possible to use cli arguments instead of positional here? 

```
ts-genapi input.api.json output.json --metadata-file metadata.json
```

Reply:
I didn't want to add option name just for the last one.  I will log an issue for refactoring to something like `--input ... --out ... --metadata-file`

Reply:
I actually like that input and output are positional - they're required aren't they? I would only make optional args named

Reply:
Yes they are required still I want to be consistent and like most of the CLI tools out there you can re-order `--out`

Reply:
Yeah, I definitely trust your judgement here - thanks for the reply!